src/Core/Layer/Layer.js

```diff
@@ -71,6 +71,8 @@ function GeometryLayer(id, object3d) {
         value: id,
         writable: false,
     });
+
+    this.postUpdate = () => {};
```

Comment:
No it's ok I get it. I forgot we transform object passed to `addLayer` to `Layer` instance anyway.